### DISCUSSION TITLE
No Temp Sensor on new AMD CPU

### DISCUSSION THREAD

**Discussion by Aenrot** [Rem0o/FanControl.Releases#2726] [General]
Hey there. I've got new pc runing and i noticed that there is no Temp sensor for my AMD Ryzen7 9700x in the list. Not really an issue but i think you should know.

**Comment by Rem0o** [comment]
See https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/issues/1418

9000 series has not been worked on yet.
